### DISCUSSION TITLE
[question] Kanji, RTF and screen

### DISCUSSION THREAD

**Discussion by Uncle-Ulty** [tsunamayo/Starship-EVO#3948] [Suggestions]
Hi Tsuna,

I'd like to ask something. Is it possible to display Kanji in STEVO screen?

I tried to copy-paste it directly, but it doesn't seem to work.

I'm building a Final Fantasy VII airship replica, The Highwind, and I intend to put the shinra kanji logo inside it, so I planned to use the screens, but the game doesn't recognize kanji. I'd like to know if it's possible to display it in any way.

thanks!

this is the original logo
![image](https://user-images.githubusercontent.com/73253472/121788319-c24e8b80-cba2-11eb-95a6-3968f235db99.png)

where the logo is supposed to be
![image](https://user-images.githubusercontent.com/73253472/121788336-d7c3b580-cba2-11eb-8f00-22a591e5ffaa.png)

![image](https://user-images.githubusercontent.com/73253472/121788342-eca04900-cba2-11eb-9558-e7a7cd0c0dc2.png)

![image](https://user-images.githubusercontent.com/73253472/121788407-6f290880-cba3-11eb-87e8-2a8340a428e9.png)

**Comment by Servomotor** [comment]
I second this. I would like to see a bunch of Unicode implemented. 
https://unicode-table.com/en/

**Reply by Uncle-Ulty** [reply]
> I second this. I would like to see a bunch of Unicode implemented.
> https://unicode-table.com/en/

Exactly! I'd love to see different symbols and characters. For instance, most cyberpunk aesthetics uses neon kanji lettering decoration. The same would work for other languages. It would enrich the game and add a plural multilanguage feeling.

**Comment by tsunamayo** [comment]
I see. Nice ship! I put this in suggestion, you guys can vote for it now!
